src/generate.rs

```diff
@@ -417,8 +464,12 @@ when_full = "block"
 [sinks.sink0]
 healthcheck = true
 inputs = ["TODO"]
```

Comment:
Is this still incomplete?

Reply:
This happens when there is no input for the component, TODO here is meant for the user to fill it up. We can either output an empty list or `["TODO"]` here.

Reply:
If it is not something for a developer to fill in, I think we should avoid the "TODO" token. That implies that there is something left incomplete within Vector. This could be something like `"source-name-goes-here"` or equivalent to indicate what kind of value it should be filled with.